code/modules/recycling/disposal.dm

```diff
@@ -133,6 +133,15 @@
 			update()
 			return
 
+		if(istype(I, /obj/item/weapon/storage/bag/chem))

```

Comment:
You should also add a contents check to this i.e. it should be possible to place the pill dispenser in the disposal should it be empty.